examples/carrier accounts/create_carrier_account.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'easypost'
+
+EasyPost.api_key = ENV['EASYPOST_TEST_API_KEY']
+
+ca = EasyPost::CarrierAccount.create(
```

Comment:
Issue: let's be as explicit as possible. `ca` should really be `carrier_account`, that is true of the couple following examples.